runtime/doc/options.txt

```diff
@@ -4872,6 +4872,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 <	Replace the ';' with a ':' or whatever separator is used.  Note that
 	this doesn't work when $INCL contains a comma or white space.
 
+							   *'plugindir'*
+'plugindir' 'pidir'	string (default "bundle")
+			global
```

Comment:
It seems to usually be to the left of tags, so `{Nvim} *'plugindir'*`, and it seems like many other tags are that way.